src/app/app-modules/associate-anm-mo/view-details/view-details.component.css

```
@@ -39,3 +39,9 @@
    cursor: not-allowed;
    pointer-events: none;
  }
+
+ .highRiskButtonColor {
+   font-family: 'Lato', sans-serif;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Lato" (font-family-name-quotes)

(font-family-name-quotes)